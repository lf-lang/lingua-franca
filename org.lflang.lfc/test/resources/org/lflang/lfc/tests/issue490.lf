# https://github.com/lf-lang/lingua-franca/issues/490
# issue is that one error ends at EOF
target Python;
main reactor R(p(3)) {
  state liss(2, 3);
  reaction (startup) {
    print(self.liss)
}
}
