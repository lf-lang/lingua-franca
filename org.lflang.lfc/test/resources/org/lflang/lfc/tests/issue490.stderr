lfc: error: Name of main reactor must match the file name (or be omitted).
--> %%%PATH.lf%%%:4:14
  |
3 | target Python;
4 | main reactor R(p(3)) {
  |              ^ Name of main reactor must match the file name (or be omitted).
  |
5 |   state liss(2, 3);

lfc: error: missing '{=' at '{'
--> %%%PATH.lf%%%:6:22
  |
5 |   state liss(2, 3);
6 |   reaction (startup) {
  |                      ^ missing '{=' at '{'
  |
7 |     print(self.liss)

lfc: error: mismatched input '<EOF>' expecting '=}'
 --> %%%PATH.lf%%%:9:2
   |
 8 | }
   | >>>>>>>>>>>>>>
 9 | }
10 | 
   | < mismatched input '<EOF>' expecting '=}'
11 | 

